bayesian/utility/db_gateway.py

```diff
@@ -88,18 +88,13 @@ def __init__(self, request_id, submit_time=None, manifest=None, deps=None):
 
     def get_request_data(self):
         """Read request data for given request id from RDS."""
-        try:
-            db_result = fetch_sa_request(rdb, self.request_id)
-            if db_result is None:
-                error_message = 'Invalid request ID {}.'.format(self.request_id)
-                logger.exception(error_message)
-                raise RDBInvalidRequestException(error_message)
-
-            return db_result
-        except Exception as e:
```

Comment:
I am not sure if this fix is going to solve the problem reported in this bug. From what I understand, the message was returned from the exception block. Now the exception block has been removed to the exception is not caught and just thrown.